examples/distributed/primitives/example_tilescale_copy.py

```
@@ -0,0 +1,195 @@
+import argparse
+import torch
+import torch.distributed as dist
+import torch.multiprocessing
+from tilelang.distributed import init_dist
+
+tilelang.disable_cache()
+os.environ['NCCL_DEBUG'] = 'WARN'  # silence NCCL log
+
+
+@tilelang.jit
+def get_kernel(M, N, block_M, block_N, threads, kernel='simt_push_tile', rank=None):
```

Ruff (0.14.2)

Unused function argument: `rank`

(ARG001)